### PR TITLE
Added link to TransactionOnFile

### PR DESCRIPTION
**Pull request by ninasyvertsen**
Connected to DX-1255

### CODE REVIEW

**Review by arebra — APPROVED** [review]
Looks good :)

File: payment-instruments/card/other-features.md [hunk 1]
```diff
@@ -63,6 +63,8 @@ when="at the 3-D Secure verification for credit card payments" full_reference=tr
 
 {% include moto.md api_resource="creditcard" %}
 
+{% include transactionOnFile.md %}
```
Comment (marinafrydenb) [inline payment-instruments/card/other-features.md:66]:
```suggestion
{% include transaction-on-file.md %}
```

File: payment-instruments/card/other-features.md [hunk 2]
```diff
@@ -115,4 +117,5 @@ payment"  %}
 [recurrence]: #recur
 [verify]: #verify
 [payout]: #payout
+[transactionOnFile]: #transactiononfile
```
Comment (arebra) [inline payment-instruments/card/other-features.md:120]:
```suggestion
[transaction-on-file]: #transactiononfile
```

**Review by marinafrydenb — APPROVED** [review]
beautiful!

**Pull request merged**